### DISCUSSION TITLE
Does D2 distinguish upper/lower case?

### DISCUSSION THREAD

**Discussion by gsgh3016** [terrastruct/d2#1126] [Q&A]
I'm trying to use D2 like this:
`
x -> A -> a
`
but it looks like this:
![d2](https://user-images.githubusercontent.com/104420634/229497914-8dec6fe5-ac6a-403a-a6a2-3d9cac849578.png)

Can I make this code to distinguish 'A' and 'a'?
If I can't, can you tell me why?

**Comment by alixander** [comment]
it's case insensitive @gsgh3016 .

You can do that by relabeling: https://play.d2lang.com/?script=qlDQtVNIBBFJmemOXCDCSsGRCxAAAP__&
